dnanexus/call_chip_from_tas.py

```diff
@@ -702,6 +706,9 @@ def main():
                 '--ctl2 %s' % (tas['rep2_ta'].get('control_id')),
             ])
 
+        if args.fragment_length:
```

Comment:
It's best to do if args.whatever is not None so we can pass other "falsey" stuff like [] and "" if necessary.  I have NOT always followed that rule in my own code, so we will probably be cleaning up things like that over time.